### DISCUSSION TITLE
Failed to add to playlist

### DISCUSSION THREAD

**Discussion by GuyWhoConquers** [libre-tube/LibreTube#6833] [General]
### Steps to reproduce

1. Click on title
2. Use option tab
3. Click "add to playlist"
4. Command response: "Failed :("
5. Important to playlist failed

### Expected behavior

Selected videos should be allowed to be added to playlist.

### Actual behavior

After trying multiple instances, videos failed to be imported into created playlist.

### LibreTube version

0.26.p

### Android version

Samsung 

### Other details

Like I have mentioned, the feature doesn't work no matter what instances I have used and I have tried turning piped proxy on and off and used other options to try adding selected videos into playlist. Th update claims that this featured was fixed after YouTube itself updated their servers an guidelines, but I have seen no improvement in thus. Everything else work fine.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
You should enable `Local streams extraction` in the instance settings and logout from any Piped account you might be logged in to.

**Reply by Bnyro** [reply]
When the instance used is IP-banned on the stream info extraction endpoint, there's no other way to use playlists.
